CHANGELOG.md

```diff
@@ -43,6 +43,12 @@ Fixes #397
 - Add support for Azure Pipelines in CI environment detection.
 - Minor fix to how Azure repository information is extracted to allow proper grouping of Azure
   repositories when various remote URLs are used to pull the repository.
+- Fixes local login on Windows.  Specifically, windows local paths are properly understood and
+  backslashes `\` are not converted to `__5c__` in paths.  Note: local login paths on Windows should be of the form:
+    1. `pulumi login --local`
+    2. `pulumi login file://~/additional/homedir/paths`
+    3. `pulumi login file://relative/path/from/current/dir`
+    4. `pulumi login file://C:/absolute/path/from/drive/root`
```

Comment:
Could we not make it so that backslashes worked here as well by calling filepath.ToSlash at some early point in the `pulumi login` path?

Reply:
good point. let me try this out to see.

Reply:
Ok. That worked.  Have updated the PR to do that.